packages/node_modules/@webex/plugin-meetings/src/roap/roap-adapter.js

```diff
@@ -95,66 +93,20 @@ const RoapAdapater = {
     }
   },
 
-  // To report Call failure and send a cancel metric most of the time ,Should be
-  // End in future
-  // TODO: discussion going on calling scrum
-  // reportCallFailure(locus, err, customToast) {
-  //   const self = this;
-  //   MeetingUtil
-  //     .getToastMessage(err)
-  //     .then((toast) => {
-  //       /* istanbul ignore else */
-  //       if (toast.sendMetrics) {
-  //         self.reportMeetingMetrics({locus, error: err, toast});
-  //       }
-
-  //       window.spark
-  //         .internal
-  //         .feature
-  //         .getFeature('developer', 'upload-call-logs')
-  //         .then((uploadLogs) => {
-  //           /* istanbul ignore else */
-  //           if (uploadLogs) {
-  //             const logs = {
-  //               locusId: locus.id,
-  //               callStart: locus.fullState.lastActive,
-  //               spaceId: locus.spaceId
-  //             };
-  //             // TODO: Remove if condition after roap compelete
-  //             /* istanbul ignore else */
-  //             // change to pass meeting
-  //             // if (locus.clientIdentifiers) {
-  //             //     logs.correlationId = locus.clientIdentifiers.correlationId;
-  //             // }
-  //             // do we need to send correlationId here?
-  //             self.submitMeetingLogs(logs);
-  //           }
-  //         });
-
-  //       // if there is an error then send joinFail and failure toast
-  //       const toastMessage = customToast || toast.toastMessage;
-  //       /* istanbul ignore else */
-  //       if (err && typeof err === 'object') {
-  //         MeetingServerActionCreators.receiveJoinFail(locus);
-  //         err.uri = locus.url;
-  //         self.sendCallFailedToast(toastMessage, locus);
-  //         config
-  //           .console
-  //           .error(`App->Locus: Join locus failed [${locus.id}][${locus.fullState.lastActive}]`, err);
-  //       }
-  //     });
-  // },
-
   sendRoap(options) {
     const {
-      roapMessage, locusId, locusSelfId, correlationId
+      roapMessage, locusId, locusSelfId, mediaId, correlationId
     } = options;
-    if (!options.mediaId) {
+
+    if (!mediaId) {
```

Comment:
what's the issue with options?

What should we define our standard to be here?

No accessing nested objects? create a temp var to represent them in all cases?, if that's the case, does it necessarily make sense to even pass an options object?